io.gitlab.Turtlico.json

```diff
@@ -97,59 +97,60 @@
                 "__pycache__",
                 "*.exe"
             ],
-            "sources" : [
-                {
-                    "type" : "archive",
-                    "url" : "https://www.python.org/ftp/python/3.8.6/Python-3.8.6.tar.xz",
-                    "sha256" : "a9e0b79d27aa056eb9cce8d63a427b5f9bab1465dee3f942dcfdb25a82f4ab8a"
-                }
-            ]
+            "sources": [{
+                "type": "archive",
+                "url": "https://www.python.org/ftp/python/3.9.9/Python-3.9.9.tar.xz",
```

Comment:
python 3.9 is in the runtime. why does it need to be built?

Reply:
I need to include tcl/tk support

Reply:
there is a way to just build tkinter without doing so.

Like this:
https://github.com/flathub/net.scribus.Scribus/blob/master/net.scribus.Scribus.yaml#L37

Reply:
Oh, I didn't know that. I will look at it.

Reply:
Another problem is that the application also needs the turtle module. Should I include the turtle.py with the manifest like it's done with tkinter?

Reply:
I made a workaround for the turtle module. It just downloads and copies the module to the site-packages directory.